gammapy/maps/axes.py

```diff
@@ -3030,6 +3030,25 @@ def to_gti(self):
             self.edges_min, self.edges_max, reference_time=self.reference_time
         )
 
+    def to_table(self, format="gadf"):
+        """Create table.
+
+        Parameters
+        ----------
+        format : {"gadf"}
+            Format specification. Default is "gadf".
+
+        Returns
+        -------
+        table : `~astropy.table.Table`
+            Table with axis data.
```

Comment:
The table contains astopy.Time object only. Is not it?

Reply:
Yes the table would contain only times, at least in the basic format

Reply:
Should we precise it explicitly? An open question...